tests/unit/mixins/sortable-item-test.js

```diff
@@ -103,8 +103,9 @@ test('registers itself with group', function(assert) {
 });
 
 test('deregisters itself when removed', function(assert) {
-  subject.remove();
-
+  run(() => {
+    subject.remove();
+  });
```

Comment:
Yeah, I was running into that failing test on my end. Not in the scope of this patch though. Let me revert that. Sorry!